pygem/unvhandler.py

```diff
@@ -34,28 +34,28 @@ def parse(self, filename):
 
 		self.infile = filename
 
-            index = -9
-            mesh_points = []
-            with open(self.infile, 'r') as input_file:
-                for num, line in enumerate(input_file):
-                    if line.startswith('  2411'):
-                        index = num
-                    if num == index + 2:
-                        if line.startswith('    -1'):
-                            break
-                        else:
-                            line = line.replace('D', 'E')
-                            l = []
-                            for t in line.split():
-                                try:
-                                    l.append(float(t))
-                                except ValueError:
-                                    pass
-                            mesh_points.append(l)
-                            index = num
-                mesh_points = np.array(mesh_points)
+		index = -9
+		mesh_points = []
+		with open(self.infile, 'r') as input_file:
+			for num, line in enumerate(input_file):
+				if line.startswith('  2411'):
+					index = num
+				if num == index + 2:
+					if line.startswith('	-1'):
```

Comment:
I think that inside startswith there are spaces, not a tab. Same below at line 85. This should solve the error in the unv tests.